src/js/components/Skeleton/__tests__/__snapshots__/Skeleton-test.tsx.snap

```diff
@@ -706,12 +706,6 @@ exports[`Skeleton with styling props 1`] = `
   -webkit-font-smoothing: antialiased;
 }
 
-@media only screen and (max-width:768px) {
```

Comment:
Why are we losing this here? I couldn't see a responsive false anywhere that would cause this to be removed? Or did I miss something

Reply:
I see the issue so.. 
```
  ${(props) => props.round && roundStyle(props.round, true, props.theme)}

```

We do need to set true because if we just have props. responsive that's undefined. hence the snap shot change because I added ` responsive && breakpoint && breakpoint.edgeSize[size];` however it was no longer going in here because responsive is not true for skeleton its undefined. StyledSkeleton is a `styledDiv` so that `responsive` prop is not pulled out from Box and is being used. The only place we have 
`  ${(props) =>
    props.round && roundStyle(props.round, props.responsive, props.theme)}` is in Box which makes sense otherwise we have `true` in other cases such as `StyledDrop` `styledTable` `styledTag` since `responsive` is not a prop on those components.

Reply:
Gotcha, thanks for clarifying. Setting `true` makes sense then in this case for backwards compatibility. Thanks for digging.